src/dev-box/build/dev-box.common.dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get -y update && \
       gawk \
       psmisc \
       python \
-      python3 \
+      python3.6 \
```

Comment:
Why using python3.6 instead of default python version

Reply:
Because some pip packages are not surport or may cause some trouble when install ansible

Reply:
A better solution is change the whole docker base image from ubuntu16.04 to ubuntu18.04